### PR TITLE
Add DigitalBrainstem.javascript-ejs-support

### PR DESCRIPTION
**Pull request by cd4d**
Extension for EJS files under MIT license

[https://marketplace.visualstudio.com/items?itemName=DigitalBrainstem.javascript-ejs-support](https://marketplace.visualstudio.com/items?itemName=DigitalBrainstem.javascript-ejs-support)

### CODE REVIEW

File: extensions.json [hunk 1]
```diff
@@ -219,6 +219,10 @@
       "id": "denoland.vscode-deno",
       "repository": "https://github.com/denoland/vscode_deno"
     },
+    {
+      "id": "DigitalBrainstem.javascript-ejs-support",
+      "repository": "https://github.com/Digitalbrainstem/ejs-grammar"
```
Comment (spoenemann) [inline extensions.json:224]: I'd suggest to specify a version to ensure we publish exactly from the sources of that release:

```suggestion
      "repository": "https://github.com/Digitalbrainstem/ejs-grammar",
      "version": "1.2.1",
      "checkout": "v1.2.1"
```

**Comment by cd4d** [comment]
Thank you for the suggestion, updated my commit. I wasn't sure if providing a version number would "freeze" it somehow and prevent updating the extension automatically in the future.

**Review by spoenemann — APPROVED** [review]

**Comment by spoenemann** [comment]
It does freeze the version, but we can update it later via https://github.com/open-vsx/publish-extensions/blob/master/upgrade-extensions.js

**Comment by jankeromnes** [comment]
Publishing worked! 🎉 https://open-vsx.org/extension/DigitalBrainstem/javascript-ejs-support

**Comment by cd4d** [comment]
Thanks @spoenemann @jankeromnes !

**Pull request merged**